### PR TITLE
175 refactor styling from stylecss into themejs

### LINKED ISSUE #175: Refactor styling from style.css into theme.js
With MUI installed, styling is now based on the `theme.js` sheet. In order to maintain consistency, consolidate the styling there and remove `style.css`.

### PR DESCRIPTION
**Pull request by masenmatthews**
---
name: Issue 175
about: removes style.css and reference to it in project's index file

---

## CONCISE description of PR 175

---
## This PR:
**1.** Removes redundant styles.css file since project is utilizing MUI

---
## Issues needing discussion/feedback (optional):
**1.** When I was about to push this originally, I thought I saw some unexpected behavior while testing manually. I'm having issues trying to reproduce this now, so additional testing/review would be appreciated.

### CODE REVIEW

**Comment by andycwilliams** [comment]
Hi masen, thanks for working on this. I don't have time to go over the code just yet but wanted to note that this should be merging into Development, not Master. You should be able to change it using the edit button at the top. Thanks!

**Review by leekahung — APPROVED** [review]
This is good.

None of the existing styles in `style.css` is currently being used at the moment and doesn't contribute to anything at the moment. It'll be fine to remove these.

File: src/index.jsx [hunk 1]
```diff
@@ -5,7 +5,6 @@ import ReactDOM from 'react-dom/client';
 // Component Imports
 import App from './App';
 // Other Imports
```
Comment (leekahung) [inline src/index.jsx:7]: We technically could remove this comment as well since it's comment was originally used for the styling import.

**Review by timbot1789 — APPROVED** [review]

**Comment by masenmatthews** [comment]
Thanks for looking this over @leekahung @andycwilliams @timbot1789 !

**Pull request merged**